packages/dashboards/src/lib/components/widget/widget-header/widget-header.component.html

```diff
@@ -27,8 +27,10 @@
 
                 <!-- link version of the header -->
                 <div *ngIf="url && !editMode">
-                    <a [href]="url" target="_blank" (mousedown)="$event.stopPropagation(); $event.preventDefault()"
-                       class="nui-text-widget nui-widget__header__content-title"
+                    <a [href]="url" (click)="changeLink($event)" target="_blank"
```

Comment:
Also, did UX really want the target to be `_blank`? I know it was `_blank` before, but it seems like for the title anchor we would want to leave how to open the page up to the user with a simple click, middle click, or right click.